### ISSUE TITLE
Create some preferences and query parameters to support customization

### ISSUE THREAD

**Issue by amanda-phet** [phetsims/mean-share-and-balance#234]
1. We'd like a way to limit the number of kicks/plates/glasses. 
Title: **Max Data** (this is vague, but I don't know of a better generic name). 
Description:  "Limit the total number of glasses, plates, or kicks." 
A number spinner makes sense to me here, with values from 2-7. 
Functionality: fairly obvious. This controls the max value in the "Number of.." spinners. 

3. We'd like a qualitative mode. 
Title:  **Qualitative Mode**. 
Description: "Encourage qualitative exploration by removing most of the numerical representations."
Toggle switch 
Functionality: 

- [ ] Level Out: Hide the tick marks checkbox 
- [ ] Distribute: Hide the total checkbox and info button  
- [ ] Fair Share: Hide the total checkbox and info button
- [ ] Balance Point: Hide the tick marks checkbox, total checkbox, and info button. Hide the tick marks and values in the grass.

**Comment by marlitas** [comment]
I posted in slack because I'm not sure if we're still doing this issue...

Earlier slack conversations:

@catherinecarter:
```
Hey team - KP and I talked with Kelly F today for about 15 minutes after our weekly Stat sim meeting regarding the “qualitative mode” in MSaB. We also asked about the max number of kicks being in the preferences. His sentiments summarized:
He didn’t see a need to make maxKicks a preference (and hence a query parameter). He said if a teacher wants students to only have 3 people, for example, they would say that not need to force the issue via a query parameter
In terms of qualitative mode specifically on the Balance Point screen:
He (like us) saw benefit to this on screens 1 and 4, but struggled to see as much meaning on screens 2 and 3
He followed his “that would be kinda cool” with, “at some point, you’d want numbers there to talk about why the mean balances where it does.”
He and KP suggested perhaps having a checkbox in the grass to “hide numbers” or “values” that you’d could check on/off to show values. This would hide both the values and the tick marks, but wouldn’t be hidden in the preferences menu.
Bottom line on the ‘qualitative mode’ was that Kelly thought it would be fine if it were solely in PhET-iO, but thought a checkbox to turn the tick marks/values on/off would be nice.
```


@kathy-phet:
```
Right - not a "mode" - just make sure you can hide everything with the visible property.
And then allow a max value for "plates/cups/balls", just as it is already done for balls.

Making these PhET-iO only, and not available with query parameters, simplifies the instrumentation quite a bit.
```

**Comment by marlitas** [comment]
In Slack @amanda-phet said:
```
Yes I think we can close that issue as something we decided against!
```

I am closing.
